src/sql-lexer/src/lexer.rs

```diff
@@ -75,10 +76,65 @@ impl LexerError {
     }
 }
 
+/// Newtype wrapper around [`String`] whose _byte_ length is guaranteed to be less than or equal to
+/// [`MAX_IDENTIFIER_LENGTH`].
+#[derive(Debug, Clone, PartialEq)]
+
```

Comment:
Did you mean to put this empty newline in?

Reply:
Whoops, removed!